app/views/layouts/_header_not_connected.haml

```diff
@@ -1,5 +1,6 @@
 %ul.nav.navbar-nav.navbar-right
-  - if display_registration_link? && !current_page?(controller: "/registrations", action: :new)
-    %li= link_to t("devise.shared.links.sign_up"), new_user_registration_path, class: "login"
-  - unless current_page?(controller: "/sessions", action: :new)
-    %li= link_to t("devise.shared.links.sign_in"), new_user_session_path, class: "login"
+  -if !open_id_context?
```

Comment:
- Style/NegatedIf: Favor `unless` over `if` for negative conditions.
- The - symbol should have one space separating it from code